src/composer.json

```diff
@@ -0,0 +1,75 @@
+{
+    "name": "blumilksoftware/example-app",
+    "type": "project",
+    "description": "example-app project.",
+    "keywords": ["framework", "laravel"],
+    "license": "MIT",
+    "require": {
+        "php": "^8.2",
+        "guzzlehttp/guzzle": "^7.7.0",
+        "inertiajs/inertia-laravel": "^0.6.9",
+        "laravel/framework": "^10.13.0",
+        "laravel/sanctum": "^3.2.5",
+        "laravel/tinker": "^2.8.1",
+        "ext-pdo": "*"
+    },
+    "require-dev": {
+        "blumilksoftware/codestyle": "^2.3.0",
```

Comment:
2.5 is current version. Please recheck all dependencies to have newest ones.

Reply:
updated